gluten-ut/spark33/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -18,12 +18,149 @@
+    .exclude("cast string to date #2")
+    .exclude("casting to fixed-precision decimals")
+    .exclude("cast from date")
+    .exclude("SPARK-32828: cast from a derived user-defined type to a base type")
+    .exclude("Fast fail for cast string type to decimal type")
+    .exclude("missing cases - from boolean")
+  enableSuite[GlutenCollectionExpressionsSuite]
+    .exclude("Map Concat")
+    .exclude("Shuffle")
+  enableSuite[GlutenComplexTypeSuite]
+    .exclude("CreateMap")
+    .exclude("MapFromArrays")
+  enableSuite[GlutenConditionalExpressionSuite]
+  enableSuite[GlutenDateExpressionsSuite]
+    // Has exception in fallback execution when we use resultDF.collect in evaluation.
+    .exclude("DATE_FROM_UNIX_DATE", "TIMESTAMP_MICROS")
+    .exclude("DayOfYear")
+    .exclude("Year")
+    .exclude("Quarter")
+    .exclude("Month")
+    .exclude("Day / DayOfMonth")
+    .exclude("DayOfWeek")
+    .exclude("extract the seconds part with fraction from timestamps")
+  enableSuite[GlutenDecimalExpressionSuite]
+    .exclude("MakeDecimal")
+  enableSuite[GlutenHashExpressionsSuite]
+    .exclude("SPARK-30633: xxHash with different type seeds")
+  enableSuite[GlutenIntervalExpressionsSuite]
+    .exclude("seconds")
+    .exclude("ANSI: extract days, hours, minutes and seconds")
+  enableSuite[GlutenJsonFunctionsSuite]
+    // Velox does not support single quotes in get_json_object function.
+    .exclude("function get_json_object - support single quotes")
+  enableSuite[GlutenLiteralExpressionSuite]
+    .exclude("default")
+    .exclude("decimal")
+    // FIXME(yma11): ObjectType is not covered in RowEncoder/Serializer in vanilla spark
+    .exclude("SPARK-37967: Literal.create support ObjectType")
+  enableSuite[GlutenMathExpressionsSuite]
+    .exclude("cos")
+    .exclude("cosh")
+    .exclude("toDegrees")
+    .exclude("toRadians")
+    .exclude("cbrt")
+    .exclude("exp")
+    .exclude("log10")
+    .exclude("log2")
+    .exclude("pow")
+    .exclude("atan2")
+  enableSuite[GlutenMiscExpressionsSuite]
+  enableSuite[GlutenNondeterministicSuite]
+    .exclude("MonotonicallyIncreasingID")
+    .exclude("SparkPartitionID")
+  enableSuite[GlutenNullExpressionsSuite]
+  enableSuite[GlutenPredicateSuite]
+    .exclude("BinaryComparison: lessThan")
+    .exclude("BinaryComparison: LessThanOrEqual")
+    .exclude("BinaryComparison: GreaterThan")
+    .exclude("BinaryComparison: GreaterThanOrEqual")
+    .exclude("SPARK-32764: compare special double/float values")
+  enableSuite[GlutenRandomSuite]
+    .exclude("random")
+    .exclude("SPARK-9127 codegen with long seed")
+  enableSuite[GlutenRegexpExpressionsSuite]
+  enableSuite[GlutenSortOrderExpressionsSuite]
+  enableSuite[GlutenStringExpressionsSuite]
+    .exclude("Substring")
+    .exclude("string for ascii")
+    .exclude("replace")
+    // ascii for Latin-1 function return from Velox is not correct
+    .exclude(("SPARK-40213: ascii for Latin-1 Supplement characters"))
```

Comment:
There is a fix PR for this UT. See https://github.com/oap-project/gluten/pull/1715. Please remove the exclusion after that PR is merged.

Reply:
FYI. The fix for ascii func. has been merged.